File: res/users/random_user/problems/1_sumNums/main.go
package main
import (
	"fmt"
	"os"
	"strconv"
)
func sumNumbers(num1 int, num2 int) int {
    fmt.Println("Hello world")
    return num1 + num2
}


func main() {
	args := [][]int{{1, 3}, {3, 5}, {7, 9}}
	expected := []int{4, 8, 16}
	file, err := os.OpenFile("user.out", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
	if err != nil {
		panic(err)
	}
	defer file.Close()
	for i := 0; i < len(args); i++ {
		value := sumNumbers(args[i][0], args[i][1])
		if value != expected[i] {
			panic(fmt.Sprintf("Expected value of %v, but got %v", expected[i], value))
		}
		_, err := file.WriteString(strconv.Itoa(value) + "\n")
		if err != nil {
			panic(err)
		}
	}
}

File: res/users/random_user/problems/1_sumNums/main.java
import java.io.BufferedWriter;
import java.lang.IllegalArgumentException;
import java.nio.file.Files;
import java.nio.file.Paths;
public class Main {
public class Solution {
    public int sumNumbers(int num1, int num2) {
        
        return num1 + num2;
    }
}

	public static void main(String[] args) throws java.io.IOException {
		int[][] argss = {{1, 3}, {3, 5}, {7, 9}};
		int[] expected = {4, 8, 16};
		Main mainClass = new Main();
		try (BufferedWriter writer = Files.newBufferedWriter(Paths.get("user.out"))) {
			for (int i = 0; i < argss.length; i++) {
				int value = mainClass.new Solution().sumNumbers(argss[i][0], argss[i][1]);
				if (value != expected[i])
					throw new IllegalArgumentException(String.format("Expected value of {%d}, but got {%d}", expected[i], value));
				writer.write(String.valueOf(value) + "\n");
			}
		}
	}
}
